### ISSUE TITLE
dragging between groups with scrolling

### ISSUE THREAD

**Issue by datom** [almende/vis#1156]
Hi,

i have many groups and want to move a item from one group to another. This works fine until the window is big enough to display all the groups at once. But when the window is smaller and i want do move a item from one group to another, its no more possible because its always scrolling and no more dragging.

For example:
http://visjs.org/examples/timeline/other/groupsPerformance.html

> run: timeline.setOptions({'height': '100px'})

now try to move i item between the groups. it always scrolling and not moving the item.

Dragging horizontal works right but vertical is not as it should be.

Regards

**Comment by josdejong** [comment]
Thanks, good point that should be fixed.

Besides that, it would be nice if the timeline automatically moves horizontally/vertically when you drag an item close to the border of the window, in case you want to drag it to some position currently not visible.

**Comment by datom** [comment]
yes that would be nice. i think on touch devices it would be very helpful

**Comment by datom** [comment]
any news when this will be fixed?

**Comment by josdejong** [comment]
I'm not sure. We're kind of busy ourselves, I can't make an estimation. 

Anyone can jump in to help out here though, it's an open source project after all :)

**Comment by thomas-barone** [comment]
Sorry for the butchered commit history!

I had the same issue, I've created a pull request #1976 for the solution that worked for me

**Comment by yotamberk** [comment]
This issue is resolved with  #1976 or with using `option.verticalScroll`

**Comment by mojoaxel** [comment]
> This issue is resolved with #1976 or with using option.verticalScroll

Sorry, but this still doesn't work.
